rdm/main.py

```diff
@@ -77,6 +80,12 @@ def parse_arguments(arguments):
     pull_parser = subparsers.add_parser('pull', help=pull_help)
     pull_parser.add_argument('config', help='Path to project `config.yml` file.')
 
+    gap_help = 'use checklist to verify documents have expected references to particular standard(s)'
+    gap_parser = subparsers.add_parser('gap', help=gap_help)
+    gap_parser.add_argument('checklist')
+    gap_parser.add_argument('files', nargs='*')
+    gap_parser.add_argument('-c', '--checklist-only', action='store_true')
```

Comment:
Why do we need a commandline argument for this? If you pass in just a checklist, and no files, won't it print the checklist only already?

Reply:
Currently it prints a warning that there are no source files. Do we want to keep this warning?

Reply:
One advantage of an command line argument for this is that it is explicit---thus users will be able to figure it out easily.

I like the idea of displaying the checklist by not providing any documents to the command because it is simple and consistent. But perhaps those "goods" don't outweigh the "good" of usability? I don't know, what do you think? Also, as you suggest, someone could accidentally not pass in any files, and without a warning they wouldn't know. One downside of the warning is that the behavior is longer consistent (since the warning is a special case).

I could see arguments either way; I would go with whatever you think.

Reply:
I am going to drop the command but add a "Note..." style warning if there are no source files. Maybe mention the no source technique in the readme.